PocketKit/Sources/PocketKit/Report/ReportRecommendationView.swift

```diff
@@ -96,30 +96,26 @@ struct ReportRecommendationView: View {
         .accessibilityIdentifier("report-recommendation")
     }
 
-    private func report(_ reason: ReportEvent.Reason) {
-        guard let url = url(for: recommendation) else {
-            return
-        }
-
-        let button = UIContext.button(identifier: .submit)
-        let content = ContentContext(url: url)
+    private func report(_ reason: ReportEntity.Reason) {
         let comment = reportComment.isEmpty ? nil : reportComment
-        let report = ReportEvent(reason: reason, comment: comment)
-        let engagement = SnowplowEngagement(type: .report, value: nil)
-        tracker.track(event: engagement, [button, content, report])
 
         isReported = true
 
         DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(1)) {
             dismiss()
         }
-    }
 
-    private func url(for recommendation: Recommendation) -> URL? {
-        recommendation.item?.resolvedURL ?? recommendation.item?.givenURL
+        guard
+            let item = recommendation.item
+        else {
+            Log.capture(message: "Reported recommendation without an associated item, not logging analytics")
+            return
+        }
+        // NOTE: As of 2/17/2023 The report view can only be called from the Home screen, so we assume that the SlateArticleReport event is the correct one.
+        tracker.track(event: Events.Home.SlateArticleReport(url: item.givenURL, reason: reason, comment: comment, itemId: item.remoteID))
```

Comment:
Call out, that the way report is setup right now, it always assumes it comes from the rec screen.